src/include/configuration/configuration_util.h

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// configuration_util.h
+//
+// Identification: src/include/configuration/configuration_util.h
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+
+#pragma once
+
+#include "configuration/configuration_type.h"
+
+namespace peloton {
+namespace configuration {
+
+class ConfigurationUtil {
+public:
+  static int32_t GET_INT(ConfigurationId id);
+  static bool GET_BOOL(ConfigurationId id);
+  static std::string GET_STRING(ConfigurationId id);
+
+  static void SET_INT(ConfigurationId id, int32_t value);
+  static void SET_BOOL(ConfigurationId id, bool value);
+  static void SET_STRING(ConfigurationId id, const std::string &value);
```

Comment:
Yes, thanks for suggestion. These functions were macros at the very beginning. I'll fix it.